### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -138,15 +136,14 @@ pushd $final_path
...
+	bundle config set --local without 'development test rmagick'
 	bundle install
-	bundle install --without development test rmagick
 	ynh_exec_warn_less bundle exec rake generate_secret_token RAILS_ENV=production
 	ynh_exec_warn_less bundle exec rake db:migrate RAILS_ENV=production
 	#ynh_exec_warn_less bundle exec rake redmine:load_default_data RAILS_ENV=production
 popd
 
-ynh_install_app_dependencies $pkg_dependencies
+#ynh_install_app_dependencies $pkg_dependencies
```
Comment (yalh76) [inline scripts/install:146]:
```suggestion
ynh_install_app_dependencies $pkg_dependencies
```
You should keep that to let remove build_dependencies

File: scripts/upgrade [hunk 2]
```diff
@@ -109,25 +108,23 @@ ynh_add_nginx_config
...
-ynh_add_config --template="../conf/configuration.yml.example" --destination="$final_path/config/configuration.yml"
+# ynh_add_config --template="../conf/database.example.yml" --destination="$final_path/config/database.yml"
+# chmod 400 "$final_path/config/database.yml"
+# chown $app:$app "$final_path/config/database.yml"
 
-chmod 400 "$final_path/config/configuration.yml"
-chown $app:$app "$final_path/config/configuration.yml"
+# ynh_add_config --template="../conf/configuration.yml.example" --destination="$final_path/config/configuration.yml"
+# chmod 400 "$final_path/config/configuration.yml"
+# chown $app:$app "$final_path/config/configuration.yml"
```
Comment (yalh76) [inline scripts/upgrade:127]: If you don't want to rewrite the configuration, you should remove all that section

File: scripts/upgrade [hunk 3]
```diff
@@ -138,13 +135,12 @@ pushd $final_path
...
 	ynh_gem install bundler passenger --no-document
-	bundle config set --local without 'development test'
+	bundle config set --local without 'development test rmagick'
 	bundle install
-	bundle install --without development test rmagick
 	ynh_exec_warn_less bundle exec rake db:migrate RAILS_ENV=production
 popd
 
-ynh_install_app_dependencies $pkg_dependencies
+#ynh_install_app_dependencies $pkg_dependencies
```
Comment (yalh76) [inline scripts/upgrade:143]:
```suggestion
ynh_install_app_dependencies $pkg_dependencies
```
You should keep that to let remove build_dependencies

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf [hunk 4]
```diff
@@ -1,5 +1,4 @@
-#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
-location __PATH__/ {
+location / {
```
Comment (yalh76) [inline conf/nginx.conf:1]:
```suggestion
#sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
location __PATH__/ {
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: conf/nginx.conf (hunk 4 above)
Comment (yalh76) [reply conf/nginx.conf:1]: We can keep the default nginx start.
The day redmine can be installed in a path, it will be already ok

Comment (ericgaspar) [reply conf/nginx.conf:1]: Redmine is not working in sub directory. #8 tries to address this issue

**Comment by yalh76** [comment]
!testme

**Review by yalh76 — APPROVED** [review]

**Review by yalh76 — APPROVED** [review]

**Pull request merged**